src/map/battle.cpp

```diff
@@ -5358,6 +5358,10 @@ static struct Damage initialize_weapon_data(struct block_list *src, struct block
 		wd.flag |= battle_range_type(src, target, skill_id, skill_lv);
 		switch(skill_id)
 		{
+			case RG_BACKSTAP:
+				if (skill_id == RG_BACKSTAP && sd && sd->status.weapon == W_DAGGER)
+				wd.div_ = -2;
+			break;
```

Comment:
This case needs to be wrapped in `#ifdef RENEWAL` since this is a renewal only feature of the skill.